clientapi/routing/routing.go

```diff
@@ -469,7 +469,7 @@ func Setup(
 		}),
 	).Methods(http.MethodPost, http.MethodOptions)
 
-	// Stub endpoints required by Riot
+	// Stub endpoints required by Element
```

Comment:
This change isn't related to this issue, but it seemed like some nice house keeping. Let me know if I should pull it out.